mappings/briefs-digital-outcomes-and-specialists-2.json

```diff
@@ -49,10 +49,10 @@
     "briefs": {
       "_meta": {
         "_": "DO NOT UPDATE BY HAND",
-        "version": "10.8.0",
+        "version": "11.0.0",
         "generated_from_framework": "digital-outcomes-and-specialists-2",
         "generated_by": "/Users/brendanbutler/digitalmarketplace/digitalmarketplace-frameworks/scripts/generate-search-config.py",
-        "generated_time": "2018-03-12T17:29:23.097131",
+        "generated_time": "2018-03-13T14:53:51.783965",
```

Comment:
🤔 that's... yesterday? (I'm sure nothing has changed since then... or has it?)

Reply:
Yes I generated it yesterday. The DOS2 mappings have no changed. I just checked by generating again and the date is the only difference.

Reply:
Cool cool cool